tensorflow/c/kernels_experimental.cc

```diff
@@ -194,6 +195,32 @@ void TF_AssignVariable(TF_OpKernelContext* ctx, int input_index,
   TF_SetStatus(status, TF_OK, "");
```

Comment:
I noticed that `TF_AssignVariable` unconditionally sets the status to `TF_OK`, so the way to check for errors is to check the status of the context. I'm not sure if it is an oversight, but this is the reason why I omitted this parameter in the ref version.

Reply:
It's probably a bug. The `copyFunc(ctx, tf_value, tf_tmp)` line should check `ctx->status()` after `copyFunc` returns. Please file a bug so we can fix `TF_AssignVariable` and `TF_AssignRefVariable` together later.

Reply:
@wangpengmit Should I change the return type for `TF_AssignRefVariable` to `TF_Status` even if it's not used then, so that it's in line with `TF_AssignVariable`?

Reply:
Oh yes please. I missed that part. We do want to change both to properly set return status later on.